### ISSUE TITLE
Conditional GET

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kiniaru/mobile-rss-reader#37]
```
What steps will reproduce the problem?
1. Download a feed that sets the HTTP header "Last-Modified"
Ex. http://pilot.prismakonsult.se/wapmet/rss.asp?ad=essa
2. Update the feed
3. Note that the server is resending the feed since the RSS client did not 
attach the header "If-Last-Modified" header

Good article:
http://fishbowl.pastiche.org/2002/10/21/http_conditional_get_for_rss_hacker
s/

The server will return status=304 as long as the client is sending the 
same If-Last-Modified


/JoP






What version of the product are you using? On what operating system?


Please provide any additional information below.


```

Original issue reported on code.google.com by `peter.n....@gmail.com` on 19 Mar 2009 at 10:39

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I've started this.  This will be on the next release candidate.  This requires 
keeping track of last-modified and ETag.

```

Original comment by `ibunto...@gmail.com` on 28 Mar 2009 at 9:29
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I've put this into Release Candidate 2.  When you use update modified or update 
all
modified, the last-modified and ETag are used.
```

Original comment by `ibunto...@gmail.com` on 22 Apr 2009 at 4:48

**Comment by GoogleCodeExporter** [comment]
```
This is fixed on version 1.11.2.
```

Original comment by `ibunto...@gmail.com` on 14 May 2009 at 5:08
- Changed state: **Fixed**
- Added labels: **Milestone-Release1.11.2**, **OpSys-All**
